magento/pwa-studio
# [PWA-2432]: Remove virtual products extension

## Description

PR to remove virtual products extension in `packages/extensions/venia-sample-virtual-products`.

## Related Issue







Closes PWA-2432

## Acceptance




See https://jira.corp.magento.com/browse/PWA-2432

### Verification Stakeholders



### Specification




## Verification Steps






#### Test scenario(s) for direct fix/feature




 1. Verify that `venia-sample-virtual-products` extension no longer appears in https://github.com/magento/pwa-studio/tree/develop/packages/extensions
2. Navigate to a Virtual Product PDP and ensure the storefront responds well. Example on develop demo instance: https://develop.pwa-venia.com/default/venia-stylist-consultation.html

#### Test scenario(s) for any existing impacted features/areas





#### Test scenario(s) for any Magento Backend Supported Configurations





#### Is Browser/Device testing needed?





#### Any ad-hoc/edge case scenarios that need to be considered?




## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)





## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
